src/components/forms/FamilyForm.tsx

```diff
@@ -202,7 +202,9 @@ export const FamilyForm = ({ family: loadedFamily }: TProps) => {
     const baseData: IFamilyFormPostBase = {
       title: formData.title,
       summary: stripHtml(formData.summary),
-      geography: formData.geography?.value || '',
+      // We still expect this value in the backend
+      geography: formData.geographies?.[0].value || '',
```

Comment:
From what I can tell we don't use this value - and we are hoping to deprecate it - so maybe we can remove it?

I haven't dug too deep to see if we use it in any children methods or functions...

Reply:
This is required by the backend (after catch up with @odrakes-cpr)